prow/config/config.go

```diff
@@ -291,6 +295,17 @@ type Branding struct {
 // PubSubSubscriptions maps GCP projects to a list of Topics.
 type PubsubSubscriptions map[string][]string
 
+// GithubOptions holds all github configuration items
+type GithubOptions struct {
+	// LinkURL configures the html link that is used in github comments,
+	// in deck, etc. default is "https://github.com"
+	LinkURL *url.URL `json:"link_url,omitempty"`
```

Comment:
These need to be:

 - `string` type, parsed on load and stored in a `*url.URL` after
 - slices to allow for fallback as we do today